runsc/boot/loader.go

```diff
@@ -201,7 +201,7 @@ type Args struct {
 }
 
 // make sure stdioFDs are always the same on initial start and on restore
-const startingStdioFD = 256
+const startingStdioFD = 600
```

Comment:
Why do we need to change this? I think it is due to vcpu fds. We can consider initializing StdioFDs  before a platform.